scripts/linters/pylint_extensions.py

```diff
@@ -2434,7 +2434,7 @@ def _process_module_tokens(
         message accordingly.
 
         Args:
-            tokens: Token. Object to access all tokens of a module.
+            tokens: TokenInfo. Object to access all tokens of a module.
```

Comment:
I think `tokens` should be List[TokenInfo].

Reply:
done